app/views/works/_contributors_table.html.erb

```diff
@@ -0,0 +1,42 @@
+<!-- Contributors section -->
+<div class="section-title">
+  <b>Additional Contributors</b>
+</div>
+
+<!--
+  Render the list of creators a hidden SPANs that are then added below
+  the HTML TABLE via JavaScript. This is so that we have a single way
+  of rendering creators already on the record *and* creators added
+  on the fly.
+-->
+<table id="contributors-table">
+  <tbody id="contributors-table-sortable">
+    <tr class="header-row">
+      <th>ORCID</th>
+      <th>Given name</th>
+      <th>Family name</th>
+      <th>Role</th>
+      <th>&nbsp;</th> <!-- move icon-->
+      <th>&nbsp;</th> <!-- delete icon-->
+    </tr>
+  </tbody>
+</table>
+
+<% if @work %>
```

Comment:
@hectorcorrea This seems odd...  I would be surprised that this form would get displayed without the work being set at least to a Work.new.  I can not imagine you adding this unless it was needed, but we may need to ticket figuring out why...

Reply:
Good catch. I removed it.